sql/snapshot.sql

```diff
@@ -0,0 +1,546 @@
+-- public.abstract_blocks definition
+
+-- Drop table
+
+-- DROP TABLE abstract_blocks;
+
+CREATE TABLE abstract_blocks (
+	hash varchar NOT NULL,
+	height int8 NOT NULL,
+	parent jsonb NULL,
+	created_at timestamp DEFAULT now() NOT NULL,
+	updated_at timestamp DEFAULT now() NOT NULL,
+	CONSTRAINT block_pkey PRIMARY KEY (hash)
+);
+
+
+-- public.abstract_transactions definition
+
+-- Drop table
+
+-- DROP TABLE abstract_transactions;
+
+CREATE TABLE abstract_transactions (
+	hash varchar NOT NULL,
+	source_addr varchar NULL,
+	destination_addr varchar NULL,
+	amount int8 NOT NULL,
+	fee int8 NULL,
+	salt int8 NULL,
+	original jsonb NULL,
```

Comment:
This field existed only for `transactions` (double check if we still need it, @marcinwadon?).
It should not exist for `fee_transactions` but it exists because of inheritance.